html/api.html

```diff
@@ -348,77 +348,104 @@ <h2 class="sub-header">
   <li><div class="tabcol0">minified</div>
     <div class="tabcol2">&nbsp;=&nbsp;&lt;true|false&gt;</div>//
     minify the result, default false, i.e. minified=true</li>
+  <li><div class="tabcol0">filter</div>
+    <div class="tabcol2">&nbsp;=&nbsp;&lt;image|video|image,video&gt;</div>//
+   filter and output the results with images, videos or both according to value.  Like "filter=video"</li>
 </ul><br/>
 <p>A search result in JSON format looks like:</p>
 <pre>
 {
-  "readme_0" : "THIS JSON IS THE RESULT OF YOUR SEARCH QUERY - THERE IS NO WEB PAGE WHICH SHOWS THE RESULT!",
-  "readme_1" : "loklak.org is the framework for a message search system, not the portal, read: http://loklak.org/about.html#notasearchportal",
-  "readme_2" : "This is supposed to be the back-end of a search portal. For the api, see http://loklak.org/api.html",
-  "readme_3" : "Parameters q=(query), source=(cache|backend|twitter|all), callback=p for jsonp, maximumRecords=(message count), minified=(true|false)",
-  "search_metadata" : {
-    "itemsPerPage" : "1",
-    "count" : "1",
-    "count_twitter_all" : 0,
-    "count_twitter_new" : 0,
-    "count_backend" : 0,
-    "count_cache" : 7336,
-    "hits" : 7336,
-    "period" : 9223372036854775807,
-    "query" : "fossasia",
-    "client" : "122.175.88.159",
-    "time" : 1196,
-    "servicereduction" : "false",
-    "scraperInfo" : "local"
+  "readme_0": "THIS JSON IS THE RESULT OF YOUR SEARCH QUERY - THERE IS NO WEB PAGE WHICH SHOWS THE RESULT!",
+  "readme_1": "loklak.org is the framework for a message search system, not the portal, read: http://loklak.org/about.html#notasearchportal",
+  "readme_2": "This is supposed to be the back-end of a search portal. For the api, see http://loklak.org/api.html",
+  "readme_3": "Parameters q=(query), source=(cache|backend|twitter|all), callback=p for jsonp, maximumRecords=(message count), minified=(true|false)",
+  "search_metadata": {
+    "startRecord": "1",
+    "maximumRecords": "1",
+    "count": "2",
+    "hits": 2,
+    "period": 81095001,
+    "query": "fossasia from:fossasia",
+    "filter": "video",
```

Comment:
json response's `filter` key will have same value as that of the query, it will be `"filter": "video, image"`